.github/workflows/mutation-testing.yml

```
@@ -105,9 +115,100 @@ jobs:
+          SCORE=$(awk -v k="$KILLED" -v t="$TOTAL" 'BEGIN { printf "%.2f", (t > 0 ? k / t * 100 : 0) }')
+          echo "total=$TOTAL" >> "$GITHUB_OUTPUT"
+          echo "killed=$KILLED" >> "$GITHUB_OUTPUT"
+          echo "score=$SCORE" >> "$GITHUB_OUTPUT"
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22 # @stryker-mutator/core ^10 requires Node >=22
+      - working-directory: web
+        run: npm ci
+      - id: frontend-score
+        working-directory: web
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:11:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)